packages/nextra-theme-docs/src/components/breadcrumb.tsx

```diff
@@ -33,9 +31,7 @@ export function Breadcrumb({
               title={item.title}
             >
               {isLink && !isActive ? (
-                <Anchor href={item.route} className="text-current no-underline">
-                  {item.title}
-                </Anchor>
+                <Anchor href={item.route}>{item.title}</Anchor>
```

Comment:
we no longer need to apply `no-underline` classes for links since:
1. there is no more global `a` class with color and underline setup
2. `@import 'tailwindcss/base';` do it for us in `styles.css`